node/utils/address.ts

```diff
@@ -4,11 +4,10 @@ export function getSelectedDeliveryAddress(
   selectedAddresses: CheckoutAddress[]
 ) {
   return selectedAddresses.find((address: CheckoutAddress) => {
-    const isSearch = address.addressType === AddressType.SEARCH
     const isPickup = address.addressType === AddressType.PICKUP
     const isInStore = address.addressType === AddressType.INSTORE
 
-    return !isSearch && !isPickup && !isInStore
```

Comment:
we should probably change the function name as well, since it doesn't return only "delivery" addresses anymore

Reply:
To `getSelectedAddres` right?

Reply:
Although I agree, I will rush the approval due to a escalation with this same problem